build-logic/src/main/kotlin/minestom.common-conventions.gradle.kts

```diff
@@ -0,0 +1,28 @@
+plugins {
+    `java-library`
+}
+
+configurations.all {
+    // We only use Jetbrains Annotations
+    exclude("org.checkerframework", "checker-qual")
+}
+
+java {
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(17))
+    }
+    sourceCompatibility = JavaVersion.VERSION_17
+    targetCompatibility = JavaVersion.VERSION_17
```

Comment:
Pretty sure you can't have both a toolchain and the source/target compatibility. I'm like 99% sure that's meant to thrown an error.